sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1563,6 +1564,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val DEFAULT_CACHE_STORAGE_LEVEL = buildConf("spark.sql.defaultCacheStorageLevel")
```

Comment:
The config's name seems effect all the cache operation in SQL. According to the change of this PR, only for `dataset.cache()` and `catalog.cacheTable()`.

What are expected if users execute `cache table`?

Reply:
`cache table` make sense to me, we should also make it work